sale_stock_available_info_popup/models/sale_order.py

```diff
@@ -13,10 +13,19 @@ class SaleOrderLine(models.Model):
         compute="_compute_immediately_usable_qty_today"
     )
 
-    @api.depends("product_id", "product_uom_qty")
+    @api.depends(
+        "product_id",
+        "product_uom_qty",
+        "product_id.immediately_usable_qty",
```

Comment:
This shouldn't be added, as only direct things that you change in your screen will provoke the recomputation, and you can't change this data through it.